### ISSUE TITLE
For each new spotlight row we create, we need to build out an indefinite amount of Color_Frames. 

### ISSUE THREAD

**Issue by HFreni** [HFreni/HFLD-Paperwork-DB#1]
Based on the schema & current associations of the Color => Spotlight through Color_Frames association, we need to allow a spotlight to have as many or as few Color Frames as the user defines. 

To do this, I want to use nested forms to 'add' or 'delete' associations through a form where we can add or subtract rows in the new/edit windows to create or remove color frames (with their associated colors) from a spotlight.

Basically, when a user creates a new spotlight, eg. Spotlight Name: Spotlight 2, we want to create a nested form where you can add/subtract rows depending as needed.

Further along the development pipeline, I'd like to add a dropdown with manufacturer_id filetering (not yet implemented).

**Comment by HFreni** [comment]
Solved in PR 2.
